components/webfield/ProgramChairConsole/SeniorAreaChairStatus.js

```diff
@@ -48,8 +48,136 @@ const SeniorAreaChairStatusRow = ({ rowData }) => (
   </tr>
 )
 
-const SeniorAreaChairStatus = ({ pcConsoleData, loadSacAcInfo }) => {
-  const { seniorAreaChairName, areaChairName } = useContext(WebFieldContext)
+const SeniorAreaChairStatusRowForDirectPaperAssignment = ({
```

Comment:
why do you have these changes in this PR?

Reply:
I'm not sure what you mean, I just merged master and resolved the conflicts. When I look at all the files it doesn't show that I made that change

Reply:
I could be wrong but what I think you're looking at is the merge commit?

Reply:
you are right, I don't see those changes in the PR now.